src/main.cpp

```diff
@@ -1,8 +1,69 @@
 #include <iostream>
+#include <algorithm>
+#include "Console.h"
+#include "boost/program_options.hpp"
 
 
-int main()
+
+namespace po = boost::program_options;
+
+int main(int argc, char **argv)
 {
-    std::cout << "Hello, World!" << std::endl;
+    // process arguments
+    po::options_description opt_desc("Allowed options");
+    opt_desc.add_options()
+            ("help", "produce help message")
+            ("create", "create new database")
+            ("upload", po::value<std::string>(), "set database path");
+
+    po::variables_map var_map;
+    po::store(po::parse_command_line(argc, argv, opt_desc), var_map);
+
+    if (var_map.count("help"))
```

Comment:
all this commands should be realized in Console, when we call our program, we can only enter the pass to database file

Reply:
> all this commands should be realized in Console, when we call our program, we can only enter the path to database file

Also removed